spec/example-unnamed-reifier.ttl

```diff
@@ -0,0 +1,6 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
```

Comment:
missing:
```
@prefix wd: <http://www.wikidata.org/entity/> .
@prefix dcterms: <http://purl.org/dc/terms/> .
```

Reply:
As per above. (Although if standalone examples are preferred by most; I'll switch to that.)